packages/market-service/src/foxy/foxy.ts

```diff
@@ -81,9 +83,9 @@ export class FoxyMarketService implements MarketService {
       const supply = foxyTotalSupply
 
       return {
-        price: marketData.priceUsd,
+        price: cgData.price,
         marketCap: '0', // TODO: add marketCap once able to get foxy marketCap data
```

Comment:
Do we not want to use FOX marketCap and total volume as well?

Reply:
or at least we can calc marketCap based on fox price

Reply:
We don't, those are two different assets. The price and price history is something we can share given FOXy is effectively "pegged" to FOX, but the rest is asset-specific

Reply:
Yeah that works with me